src/compass/download_orbits.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import requests
+import re
+import os
+import glob
+import argparse
+import datetime
+from html.parser import HTMLParser
+
+server = 'https://scihub.copernicus.eu/gnss/'
+
+orbitMap = [('precise', 'AUX_POEORB'),
+            ('restituted', 'AUX_RESORB')]
+
+datefmt = "%Y%m%dT%H%M%S"
+queryfmt = "%Y-%m-%d"
+queryfmt2 = "%Y/%m/%d/"
+
+#Generic credentials to query and download orbit files
+credentials = ('gnssguest', 'gnssguest')
+
+
+def cmdLineParse():
+    '''
+    Command line parser.
+    '''
+
+    parser = argparse.ArgumentParser(description='Fetch Sentinel-1 orbits')
```

Comment:
May be clarify the logic with which the orbits are downloaded, i.e. look for POE first if POE is not found download the ROE?